Orange/widgets/gui.py

```diff
@@ -1195,6 +1195,36 @@ def attributeItem(var):
     return attributeIconDict[var], var.name
 
 
+class ListViewWithSizeHint(QListView):
+    def __init__(self, *args, preferred_size=None, **kwargs):
+        super().__init__(*args, **kwargs)
```

Comment:
I meant in above case when setting sizeHint from kwarg. This makes it inconvenient compared to tuples, but is a rather small price to pay if it gets you rid of a whole class. :smiley:  

> "It is better to have 100 functions operate on one data structure than 10 functions on 10 data structures."
> — Alan Perlis